css/main.css

```diff
@@ -0,0 +1,1623 @@
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 200px;
+  width: 90%;
+  font-size: 20px;
+  line-height: 20px;
+  margin-top: 0;
+  margin-bottom: 30px;
+}
+
+.section1-3-heading {
+  font-weight: 700;
+  letter-spacing: -0.84px;
+  color: #fff;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  gap: normal;
+  position: relative;
+  top: 32px;
+  margin-left: 6px;
+  line-height: 20px;
+  margin-top: 50px;
+  margin-bottom: 30px;
+}
+
+.media-align-2 {
+  margin-top: 0;
+}
+
+.media-align-3 {
+  margin-top: 0;
+}
+
+.section1-3-heading-title-link {
+  text-decoration: none;
+  color: #fff;
+  position: relative;
+  top: 75px;
+  font-weight: lighter;
+  font-size: 20px;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  margin-left: 6px;
+}
+
+#ai {
+  margin-left: 6px;
+  position: relative;
+  bottom: 10px;
+}
+
+.section1-grid {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr 1fr;
+}
+
+#tnw-logo {
+  width: 20%;
+  height: 10%;
+  position: relative;
+  margin: 9px;
+  top: -75px;
+}
+
+.main-container {
+  margin: 10px 0 0 0;
+  width: auto;
+  height: auto;
+}
+
+.link {
+  text-decoration: none;
+  color: rgb(255, 86, 34);
+  font-size: 20px;
+  font-weight: 700;
+}
+
+.link-2 {
+  text-decoration: none;
+  color: rgb(255, 86, 34);
+  font-size: 14px;
+  height: 31px;
+  margin-top: 30px;
+  margin-right: 23px;
+  margin-bottom: 0;
+}
+
+.f1-grid {
+  display: block;
+}
+
+.f1 {
+  width: 100%;
+  height: 100px;
+  line-height: 0;
+}
+
+.f1-img {
+  width: 75px;
+  height: 75px;
+}
+
+.f1-media {
+  margin-left: 100px;
+  margin-top: -80px;
+}
+
+.f1 h4 {
+  margin: 0 0 0 8px;
+}
+
+.f1-flex {
+  display: flex;
+  margin: 10px 0 0 8px;
+}
+
+.f1-by {
+  text-decoration: none;
+  color: rgb(85, 85, 85);
+  font-size: 8px;
+}
+
+.f1-flex span {
+  color: rgb(85, 85, 85);
+  font-size: 8px;
+}
+
+.gallery-link {
+  text-decoration: none;
+  color: black;
+  font-weight: 700;
+  font-size: 10px;
+  text-align: left;
+  line-height: 20px;
+}
+
+.flex {
+  display: block;
+  justify-content: space-between;
+}
+
+.f2-grid {
+  display: block;
+  margin-bottom: 0;
+}
+
+.f2 {
+  width: 294px;
+  height: 288px;
+  border: 1px solid transparent;
+}
+
+.align {
+  position: relative;
+  top: 0;
+  height: 287px;
+}
+
+.f2 a {
+  text-decoration: none;
+  position: relative;
+}
+
+.f2-title {
+  background-color: rgb(255, 68, 34);
+  color: white;
+  width: 270px;
+  height: 39px;
+  font-size: 18px !important;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  padding-top: 10px;
+  text-align: center;
+  position: relative;
+  top: -25px;
+}
+
+.f2-img {
+  width: 60px;
+  height: 60px;
+  position: absolute;
+  top: 10px;
+  left: 18px;
+  z-index: 10;
+}
+
+.f2-seccontainer {
+  background-color: rgba(255, 68, 34, 0.08);
+  width: 270px;
+  height: 45px;
+  font-size: 14px;
+  line-height: 21px;
+  position: absolute;
+  top: 50px;
+}
```

Comment:
- [ ] To make this full width in **mobile** use the following (the reason we cannot use the automatic width is because this has a `position: absolute;`
```
.f2-seccontainer {
    background-color: rgba(255, 68, 34, 0.08);
    width: calc(100vw - 36px);
    height: 45px;
    font-size: 14px;
    line-height: 21px;
    position: absolute;
    top: 50px;
}
```